src/map_storage/jwt.py

```diff
@@ -45,7 +44,7 @@ def decode_jwt():
         try:
             _, token = auth.split(' ', 1)
             g.jwt_claims = jwt.decode(
-                token, app.config['JWT_PUBLIC_KEY'], app.config['ALGORITHM'])
+                token, app.config['JWT_PUBLIC_KEY'], 'RS512')
```

Comment:
How come the change?

Reply:
This was left over from the IAM service which is the service that defines the algorithm in the settings. It should rather be extracted from the public key here I suppose, I'll append a fix for that.